docs/output.md

```diff
@@ -59,6 +62,23 @@ Contains FastQ files with quality and adapter trimmed reads for each sample, alo
 
 Single-end data will have slightly different file names and only one FastQ file per sample.
 
+## SortMeRNA
+When `--remove_rRNA` is specified, nfcore/rnaseq pipeline uses [SortMeRNA](https://github.com/biocore/sortmerna) for removal of rRNA. SortMeRNA requires reference sequences and these are by default from the [SILVA database](https://www.arb-silva.de/).
+
+MultiQC reports currently do not contain statistics of the rRNA removal.
```

Comment:
Aww, sad times! 😢

Reply:
Something we'll add soon - I guess you have enough PRs on MultiQC for a while to play with 😄

Reply:
Found the module, adding in a separate small PR.